.github/workflows/php.yml

```diff
@@ -19,6 +19,7 @@ jobs:
         uses: shivammathur/setup-php@v2
         with:
           php-version: '8.2'
+          coverage: xdebug
```

Comment:
This is just for "convenience". There could also be a tighter integration (I think, but I have never done that).